components/suggest/src/suggestion.rs

```diff
@@ -96,13 +96,35 @@ pub enum Suggestion {
         icon: Option<Vec<u8>>,
         icon_mimetype: Option<String>,
         score: f64,
+        match_info: FtsMatchInfo,
     },
     Exposure {
         suggestion_type: String,
         score: f64,
     },
 }
 
+/// Additional data about how an FTS match was made(https://bugzilla.mozilla.org/show_bug.cgi?id=1931373)
```

Comment:
Nit: Missing a space in "made(" but I'd just leave out the bug URL. If we need to find the bug/PR where this change was made, we can check blame.